core/runtime/jsruntime.rs

```diff
@@ -840,6 +841,23 @@ impl JsRuntime {
     let mut esm_entrypoints = vec![];
 
     futures::executor::block_on(async {
+      if self.init_mode == InitMode::New {
+        let js_files = include_js_files!(
+          core
+          "00_primordials.js",
+          "01_core.js",
+          "02_error.js",
```

Comment:
Making this inline extension will not print `rerun-if-changed:` directive which will not rebuild core once these JS files change. Can you fix that?